fv3core/stencils/saturation_adjustment.py

```diff
@@ -618,13 +544,22 @@ def wqs1_fn_2(it, ap1, ta, den):
 
 
 @gtstencil()
-def wqs2_stencil_w(ta: sd, den: sd, wqsat: sd, dqdt: sd):
+def wqs2_stencil_w(
+    ta: FloatField, den: FloatField, wqsat: FloatField, dqdt: FloatField
+):
     with computation(PARALLEL), interval(...):
         wqsat, dqdt = wqs2_fn_w(ta, den)
```

Comment:
I think we can delete this now? is anything using it anymore?

Reply:
I didn't find anything using it, so I can delete it.